src/components/Notifier/index.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withSnackbar } from 'notistack'
+import actions from './actions'
+import selector from './selector'
+
+class Notifier extends Component {
```

Comment:
Not necessary to change right now, but let's write new code using modern React (hooks, react.memo, etc)